modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,68 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+The following annotations are supported by Knative Serving:
+
+* Disabling routes, which is specified by the `serving.knative.openshift.io/disableRoute` annotation.
+* Enabling pass-through, which is specified by the `serving.knative.openshift.io/enablePassthrough` annotation.
```

Comment:
This general paragraph would fit better at the top of the page when we can enumerate all supported annotations like disableRoute, enablePassthrough or cluster local one.

Currently, this fragment is included as second one.

Reply:
I'll move it